udata/core/dataset/rdf.py

```diff
@@ -488,6 +489,7 @@ def dataset_from_rdf(graph, dataset=None, node=None):
     default_license = dataset.license or License.default()
     dataset_license = rdf_value(d, DCT.license)
     dataset.license = License.guess(dataset_license, *licenses, default=default_license)
+    dataset.access_rights = rdf_value(d, DCT.accessRights)
```

Comment:
We should map from and towards the European vocabulary: http://publications.europa.eu/resource/authority/access-right.